app/colorCoder.js

```diff
@@ -1,44 +1,29 @@
-
-const MajorColorNames = [
-    "WHITE", "RED", "BLACK", "YELLOW", "VIOLET"
-];
-const MinorColorNames = [
-	"BLUE", "ORANGE", "GREEN", "BROWN", "SLATE"
-];
-
 function ColorPair(){
            this.majorColor;
            this.minorColor;
 }
-
 ColorPair.prototype.toString=function(){
 	return `MajorColor:${this.majorColor},MinorColr:${this.minorColor}`;
 }
-
-function getColorFromPairNumber(pairNumber)
-{
-	let minorSize = MajorColorNames.length;
-	let majorSize = MinorColorNames.length;
+function getColorFromPairNumber(pairNumber){
+	const minorSize = MajorColorNames.length;
+	const majorSize = MinorColorNames.length;
 	
-	if (pairNumber < 1 || pairNumber > minorSize * majorSize)
- 	{
- 		throw `Argument PairNumber:${pairNumber} is outside the allowed range` 
-	}
-	let zeroBasedPairNumber = pairNumber - 1;
-    let majorIndex = parseInt (zeroBasedPairNumber / minorSize);
-    let minorIndex = parseInt(zeroBasedPairNumber % minorSize);
+	if (pairNumber < 1 || pairNumber > minorSize * majorSize) {
```

Comment:
Think how you could prove this range-check. What change would be required in the code to make it testable?